src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
Damn your brain! Now I think about it really like that you can declare all properties within a class as lazy. That's a really neat way to do it and way better than what I was thinking.